File: JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL3.py
import math
Ln_valor=4
Ln_result=Ln_valor%2

#presentar si un número dado es par o impar
if (Ln_result==0):
    print("El número es par")
else:
    print("El número no es par")


print("-----------------------------------------------------")

## recuperar valor entero de un int
Ln_valor =28.129
Ln_parte_entera= int( Ln_valor//1)
print("parte entera recuperada: ",Ln_parte_entera)
print("-----------------------------------------------------")

## recuperar parte decimal de un int
print("recuperando parte decimal")
Ln_parte_decimal= round((Ln_valor%1),2)
print("redondeado: ",Ln_parte_decimal)

print("-----------------------------------------------------")

Ln_parte_decimal= Ln_valor%1
Ln_parte_decimal= '%.3f'%(Ln_parte_decimal)
print("truncado: ",Ln_parte_decimal)


print("-----------------------------------------------------")
Ln_valor=80.1265
print("El número es:", Ln_valor)
Ln_parte_entera,Ln_parte_decimal=math.modf(Ln_valor)

print(f"su parte entera es {round(Ln_parte_entera,4)}, y su parte decimal es: {Ln_parte_decimal}")

File: talleres en js/JOSÉ_ALEJANDRO_GÓMEZ_PAREDES_TLL3.js
let Ln_Valor = 6;
let Ln_Result = Ln_Valor % 2;
console.log("--------------------------------------------------------");
console.log("presentar por pantalla si es par o no");
console.log(`El número ${Ln_Valor} es par?: `);
if (Ln_Result == 0) {
  console.log("si es par");
} else {
  console.log("no es par");
}
console.log("--------------------------------------------------------");

console.log("Recuperando el valor de un número entero");
Ln_Valor = 28.129;
let Ln_Parte_Entera = Math.floor(Ln_Valor);
console.log(Ln_Parte_Entera);
console.log("");

console.log("Recuperando la parte entera en una variable int");
Ln_Parte_Entera = parseInt(Ln_Valor);
console.log(Ln_Parte_Entera);
console.log("");

console.log("Recuperando la parte decimal");
let Ln_Parte_Decimal = Ln_Valor - Math.floor(Ln_Valor);
console.log(Ln_Parte_Decimal);
console.log("");

console.log("Redondeo");
Ln_Parte_Decimal = Math.round((Ln_Valor - Math.floor(Ln_Valor)) * 100) / 100;
console.log(Ln_Parte_Decimal);
console.log("");
console.log("Truncar con Math.trunc");
Ln_Parte_Decimal = Math.trunc(Ln_Valor);
console.log(Ln_Parte_Decimal);
console.log("");

console.log("Truncar con Math.floor");
Ln_Valor = 80.1265;
Ln_Parte_Entera = Math.floor(Ln_Valor);
Ln_Parte_Decimal = Ln_Valor - Ln_Parte_Entera;
console.log(Ln_Parte_Decimal.toFixed(4));
console.log("");
